firmware_mod/scripts/detectionOn.sh

```diff
@@ -4,6 +4,8 @@
 . /system/sdcard/config/motion.conf
 . /system/sdcard/scripts/common_functions.sh
 
+include /system/sdcard/config/telegram.conf
```

Comment:
@jmtatsch here is the benefit of including this way. Users may not have renamed `telegram.conf.dist` to `telegram.conf`. `detectionOn.sh` will fail in that case.